ramlfications/models/data_types.py

```diff
@@ -107,6 +107,23 @@ class RAMLDataType(object):
 
 
 @attr.s
+class Example(object):
```

Comment:
I wonder if there should be a ``ramlfications.models.examples`` module for this, then: data types can also have examples, not only parameters.